Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx

```diff
@@ -270,7 +276,8 @@ const AlpideRespSimMat* AlpideSimResponse::getResponse(float vRow, float vCol, f
   } else {
     flipRow = true;
   }
-  if (vRow > mRowMax) {
+  if (vRow > rowMax) {
+    std::cout << "FAIL 3 " << vRow << "\t" << mRowMax << std::endl;
```

Comment:
please use `LOG(ERROR)`

Reply:
Thanks for spotting this, it was a debug cout that we did not remove!